### PR TITLE
Add pks-login command to Hammer

### PR DESCRIPTION
**Pull request by gmrodgers**
A new command for logging into PKS environments.  The only new required field in the environment file is the PKS API URL.  I put in the URL as otherwise, I would need to differentiate between the staged URL and the templated one (the field is configurable in Opsman).  Thoughts/Comments welcome.  The structure is very similar to cf-login (some would say a copy-pasta)...

### CODE REVIEW

File: environment/config.go [hunk 1]
```diff
@@ -103,9 +115,26 @@ func newLockfile(data environmentReader) (Config, error) {
...
+		},
 	}, nil
 }
+
+func parseUrl(urlToParse string, data environmentReader) (*url.URL, net.IP, error) {
+	parsedURL, err := url.Parse(urlToParse)
+	if err != nil {
+		return nil, nil, err
+	}
+	ip := net.ParseIP(data.IP)
```
Comment (edwardecook) [inline environment/config.go]: Not entirely sure why the ip is being passed here.

File: go.mod [hunk 2]
```diff
@@ -3,15 +3,74 @@ module github.com/pivotal/hammer
 go 1.12
 
 require (
-	github.com/golang/protobuf v1.3.1 // indirect
+	cloud.google.com/go v0.43.0 // indirect
```
Comment (edwardecook) [inline go.mod]: I think you need to run `go mod tidy`

**Comment by pivotal-issuemaster** [comment]
@gmrodgers Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=pivotal/hammer&pullRequestId=6) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@gmrodgers Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=pivotal/hammer&pullRequestId=6)!

**Pull request merged**